web/app/app.js

```diff
@@ -18,7 +18,8 @@
         'ngFileSaver',
         'snap',
         'tmh.dynamicLocale',
-        'pascalprecht.translate'
+        'pascalprecht.translate',
+        'as.sortable'
```

Comment:
Still confident we can implement a sortable using just gridster and avoid a new dependency, I'll have a look ;)